lib/extension.ts

```
@@ -250,25 +251,152 @@ export const setupPostcodeLookup = (config: Config, targets: OutputFields, pageT
+        for (const key in contexts) {
+          if (key === countryValue.toUpperCase()) {
+            contextDetails = contexts[key];
+            break;
+          }
+        }
+          
+        // If no match by ISO3, try to match by name
+        if (!contextDetails) {
+          for (const key in contexts) {
+            const context = contexts[key];
+            if (context && context.description && 
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)